app/views/spam2/_spam.html.erb

```diff
@@ -0,0 +1,109 @@
+<script>
+$(document).ready(function() {
```

Comment:
Please keep the style consistent and format this block & add some comments.

Reply:
Formatted and added a few comments
Thanks!